DGC-all-schemas-combined.json

```diff
@@ -0,0 +1,390 @@
+{
+  "title": "EU DGC",
+  "$defs": {
+    "dob_valid_date_range": {
+      "description": "Date type ISO 8601 - date part only, restricted to range 1900-2099",
+      "type": "string",
+      "pattern": "[19|20][0-9][0-9]-(0[1-9]|1[0-2])-([0-2][1-9]|3[0|1])"
```

Comment:
Se https://github.com/ehn-digital-green-development/ehn-dgc-schema/pull/8

Reply:
The use of proper JSON formats eases impementation. Additional constraints using pattern is possible.

Reply:
Nice - lets merge that in.